notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -51,7 +51,15 @@
    "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from pathlib import Path
</pre><p>cells  order looks wrong, you trying to load model, run it's inference and only after that calibrate. It may lead to getting invalid result on this step for first running. maybe include this step into step when we convert model instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>About cell order agree, moved calibration cell up.</p>